kolibri/core/assets/src/views/content-renderer/index.vue

```diff
@@ -118,36 +113,29 @@
     },
     created() {
       this.updateRendererComponent();
-      // This means this component has to be torn down on channel switches.
       this.$watch('files', this.updateRendererComponent);
     },
     data: () => ({
       currentViewClass: null,
       noRendererAvailable: false,
     }),
     methods: {
-      /**
```

Comment:
Have reinstated all the comments locally - a bit confused why they got removed, as it seemed to be a bit selective, as some comments were left intact. Not because of prettier though, it was due to the code I wrote to change the components. Will double check that code for future purposes, but it shouldn't be necessary after this.